src/styles/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &--4 &__star:nth-child(-n+4) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
What if user wants to put 5 stars or 1 ? Copy and paste the logic from Stars task. You can also implement it with Sass loop.